### ISSUE TITLE
Charts which need to be scrolled (both vertically and horizontally) are not keyboard accessible  

### ISSUE THREAD

**Issue by gouravnema** [recharts/recharts#3871]
- [x] I have searched the [issues](https://github.com/recharts/recharts/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://26vmpy.csb.app](https://26vmpy.csb.app)

### Steps to reproduce
1. Use keyboard tab key to navigate charts which are not present in view port and needs scrolling. 
2. Use down arrow and side arrow key to hover tabstoppoints of charts which you need to scroll

### What is expected?
Charts should show tooltip when keyboard down and right/left is pressed as it is happening to the charts which are present in viewport.  

### What is actually happening?
Only Charts in viewport are keyboard accessible those needs scrolling are not accessible.

| Environment | Info |
|---|---|
| Recharts | v2.9.0 |
| React | react |
| System | Mac OS and Windows  |
| Browser | Chrome |

---
https://codesandbox.io/s/focused-butterfly-26vmpy   Pls check code for above sandbox link here for more details on implementation.

**Comment by gouravnema** [comment]
I am happy to submit PR. If team accepts this as an bug, but don't have bandwidth to push a fix for it.

**Comment by ckifer** [comment]
@julianna-langston just tagging as an FYI (she implemented the accessibilityLayer feature)

@gouravnema please, that would be amazing! Not sure why this happens but the fix should be somewhere in AccessibilityManager

**Comment by gouravnema** [comment]
It is happening because we are not adding scroll offset. This update is fixing it. Give me a day or two to finalize my pr.  https://github.com/recharts/recharts/commit/b5358d0a1616921ac470e8369fe10ebef38bc87b

**Comment by gouravnema** [comment]
> Yeah this makes sense since the arrow keys control the horz scroll bar :/. I don't think its an issue vertically though?
> 
> @julianna-langston just tagging as an FYI (she implemented the accessibilityLayer feature)
> 
> @gouravnema please, that would be amazing! Not sure why this happens but the fix should be somewhere in AccessibilityManager

It is there in vertical alignment as well. Also nothing to do with arrow hey. SpoofMouse function is passing coordinates relative to screen but tooltip calculations are happening for the coordinates of actual position of element in DOM.

**Comment by ckifer** [comment]
Sweet no worries!

**Comment by gouravnema** [comment]
@ckifer when we can expect this fix to be released ?

**Comment by ckifer** [comment]
The next minor release. Unfortunately I can't really promise anything exact due to other priorities/lack of capacity - probably within the next 1-3 weeks.

**Comment by nikolasrieble** [comment]
Released with https://github.com/recharts/recharts/releases/tag/v2.10

**Comment by msftedad** [comment]
Hi @nikolasrieble ,please find the below comments from PO team as issue is still repro'ing in 2.10 version
Here's the remarks from dependent team:
"I updated to v2.10 and tested locally, the issue still persists.
I might be looking at the wrong list but I don't see the accessibility issue listed in the release note:
https://github.com/recharts/recharts/releases/tag/v2.10"

**Comment by ckifer** [comment]
In the release notes: "Added scroll offset positions to spoof calculations to fix accessibility issue"
